internal/jobcontainers/path_test.go

```diff
@@ -31,52 +33,159 @@ func TestSearchPath(t *testing.T) {
 	}
 }
 
-func TestGetApplicationName(t *testing.T) {
+func TestGetApplicationNamePing(t *testing.T) {
 	expected := "C:\\WINDOWS\\system32\\ping.exe"
 
 	cwd, err := os.Getwd()
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	path, _, err := getApplicationName("ping", cwd, os.Getenv("PATH"))
+	pathEnv := os.Getenv("PATH")
+
+	path, _, err := getApplicationName("ping", cwd, pathEnv)
+	if err != nil {
+		t.Fatal(err)
+	}
+	assertStr(t, expected, path)
+
+	path, _, err = getApplicationName("./ping", cwd, pathEnv)
 	if err != nil {
 		t.Fatal(err)
 	}
 	assertStr(t, expected, path)
```

Comment:
Is this saying a lookup for `./ping` should resolve to `C:\Windows\system32\ping.exe`? That would only be true if our working directory is `C:\Windows\system32`, right?

Reply:
I think the problem is we're treating a lookup for `./ping` as equivalent to `ping`. The former should _only_ return a result in the working directory, otherwise fail. The latter should look in other search paths as well.

Reply:
Good catch, this is incorrect behavior